### ISSUE TITLE
Develop homepage

### ISSUE THREAD

**Issue by james-minton** [Hyperobjekt/seeing-solitary#42]
Please refer to [mockups here](https://www.figma.com/file/JrvMTSccXdO97taW5dWK62/Yale-Seeing-Solitary-HP?node-id=863%3A7139).

Notes:

- Refer to comments in mockup for notes about individual elements.
- Many of these comments refer to animations. Consider these "nice to have" and please implement this page first as a static design. We will circle back to animations once this "static" phase is done. That said, please code these elements in a way that will make animations easier to implement.
- Ensure that type is encoded using semantic tags (h1, h2, p, etc.). This is necessary for accessibility and other reasons. Styles for these tags can be configured in `tailwindsconfig.js` -- do not encode text styles inline, save for exceptional cases.

This ticket links with [#43](https://github.com/Hyperobjekt/seeing-solitary/issues/43) re: establishing a framework for animations.
